### PR TITLE
section commit B for July 6th, '20

### PR DESCRIPTION
**Pull request by croqueGrec09**
refactoring and merger done for LicenseProperty

### CODE REVIEW

File: app/grails-app/views/subscription/_properties.gsp [hunk 1]
```diff
@@ -19,11 +19,11 @@
 <g:if test="${subscriptionInstance.getCalculatedType() in [CalculatedType.TYPE_CONSORTIAL,CalculatedType.TYPE_ADMINISTRATIVE]}">
     <div class="ui card la-dl-no-table ">
         <div class="content">
-            <h5 class="ui header">${message(code:'subscription.properties.consortium')}</h5>
+            <h5 class="ui header">${message(code:'license.properties.consortium')}</h5>
             <div id="member_props_div">
                 <g:render template="/templates/properties/members" model="${[
                         prop_desc: PropertyDefinition.SUB_PROP,
-                        ownobj: subscriptionInstance,
+                        ownobj: license,
```
Comment (klodav) [inline app/grails-app/views/subscription/_properties.gsp:26]: license is here correct?

Comment (croqueGrec09) [reply app/grails-app/views/subscription/_properties.gsp]: No, it is not, fixed in d4429dd

File: app/migrations/changelog-2020-07-06.groovy [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+			change {
+				sql.execute("ALTER TABLE license_custom_property RENAME last_updated_cascading  TO lp_last_updated_cascading;")
+			}
+			rollback {}
+		}
+	}
+
+	changeSet(author: "galffy (modified)", id: "1594019505211-18") {
+		addColumn(schemaName: "public", tableName: "license_custom_property") {
+			column(name: "lp_is_public", type: "text")
```
Comment (klodav) [inline app/migrations/changelog-2020-07-06.groovy:148]: type: "text" ?

Comment (croqueGrec09) [reply app/migrations/changelog-2020-07-06.groovy]: Is wrong. Should be bool. Finxed in 8ad8e8f

**Pull request merged**